bolt-sidecar/Cargo.toml

```diff
@@ -24,21 +23,20 @@ blst = "0.3.12"
 tree_hash = "0.5"
 tree_hash_derive = "0.5"
 secp256k1 = { version = "0.29.0", features = ["rand"] }
-ssz_rs = { git = "https://github.com/ralexstokes/ssz-rs" }
+ssz_rs = { git = "https://github.com/ralexstokes/ssz-rs", rev = "ec3073e" }
 ethereum_ssz = "0.5"
-ethereum_ssz_derive = "0.5"
 
 # alloy
-alloy = { version = "0.6.3", features = [
+alloy = { version = "0.6.4", features = [
   "full",
```

Comment:
Wonder if we need `full` here

Reply:
if we remove it we'll have to add a bunch of other features in its place, but I haven't tried